plugins/orchestrator/README.md

```diff
@@ -225,7 +225,7 @@ The plugin provides OpenAPI `v2` endpoints definition to facilitate communicatio
 In addition, by leveraging on OpenAPI spec, it is possible to generate clients and create CI steps.
 
 OpenAPI specification [file](https://github.com/janus-idp/backstage-plugins/blob/main/plugins/orchestrator-common/src/openapi/openapi.yaml) is available in [orchestrator-common](https://github.com/janus-idp/backstage-plugins/blob/main/plugins/orchestrator-common).  
-OpenAPI specification documentation is available [here](https://github.com/janus-idp/backstage-plugins/blob/main/plugins/orchestrator-common/src/generated/docs/markdown/README.md)
+OpenAPI specification documentation is available [here](https://github.com/janus-idp/backstage-plugins/blob/main/plugins/orchestrator-common/src/generated/docs)
```

Comment:
I checked and github won't present you an option to show index.html, it only previews .md files
So with this change navigating to the api docs will give you this
![image](https://github.com/user-attachments/assets/d01b2529-7944-4dc2-bf54-9f8848317439)
which is not user friendly
I suggest to keep the old link

Reply:
and opening html/index.html will give me this
![image](https://github.com/user-attachments/assets/876f8297-ff2f-45a3-bbde-67b65698be64)

in short, we can't mix md viewer with html viewing
in website and github we need to show md
url should return html

Reply:
i fixed everything